index.html

```diff
@@ -3221,6 +3224,117 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+    <div class="role" id="label">
+      <rdef>label</rdef>
+      <div class="role-description">
+        <p>A visible name or caption for a user interface component.</p>
+        <p>A label element can provide an accessible name for a user interface component if it is programatically associated with the element. Authors MAY associate a label element with the element by using one of two methods:</p>
```

Comment:
Wordsmithing: instead of "a label element", perhaps "An element with role `label`" (that said, we do seem to do both).